pipelines/participant_demographics/run.py

```diff
@@ -10,7 +10,13 @@
 from . import prompts
 from .clean import clean_predictions
 
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```

Some merge conflict leftover here